src/hardware/mixer.cpp

```diff
@@ -676,14 +658,15 @@ static void MIXER_MixData(Bitu needed) {
 	mixer.done = needed;
 }
 
+static void MIXER_QueueAudio(Bitu);
```

Comment:
Let's avoid bitu.
Suggest the smallest type that captures the largest possible length we would get. The other audio code ises uint16's because we never expect more than 16k; the less wiggle room the better (if things go off the rails or someone is trying to attack the code or get it read or write values .. then the most they'll get is up to 32k.